Demo/MozillaSocial-iOS/MoSoContent/Auth.swift

```diff
@@ -0,0 +1,138 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import AuthenticationServices
+
+let isProd = false
+let baseURL = isProd ? "https://mozilla.social" : "https://stage.moztodon.nonprod.webservices.mozgcp.net"
+let redirectScheme = "mozillasocial"
+let redirectURI = redirectScheme + "://auth"
+
+class Auth: NSObject, ASWebAuthenticationPresentationContextProviding {
```

Comment:
do you think we can use a more explicit name for this type, like `AuthenticationService` or similar?